docs/Releases/release110.md

```diff
@@ -18,7 +18,7 @@ improvements.
   connect to Apache Kafka clusters.
 - Enabled JWT-based authentication: Enabled JWT-based authentication
   to connect to Klaw Cluster API. You can configure
-  klaw.clusterapi.access.base64.secret in both core and cluster-api
+  Klaw.clusterapi.access.base64.secret in both core and cluster-api
```

Comment:
Changing spelling of "Klaw" should not be part of this PR, since it's for the rule "Klaw Core", can you reverse this one?